docs/design/storagenode-graceful-exit-ui.md

```diff
@@ -0,0 +1,53 @@
+# Storage Node Graceful Exit - User Interface
+
+## Abstract
+
+A storage node operator needs the ability to request a Graceful Exit on a per satellite basis.
+This document describes how the storage node operator manages the Graceful Exit process.
+
+## Background
+
+The storage node operator needs:
+- a way to initiate graceful exit, 
+- a way to monitor graceful exit progress, and
+- terminate graceful exit without escrow.
+
+## Design
+
+Add a command `storagenode exit-satellite` to initiate a Graceful Exit.
+
+The command should present a list of satellites to exit. The user needs to type the satellite domain name to start exiting. Only satellites that haven't been exited will be displayed.
```

Comment:
The command should display a detailed explanation about the consequences of starting the graceful exit.

Reply:
Updated